README.md

```diff
@@ -2,55 +2,32 @@
 
 # Project Name
 
+Survey Form
+
 > Description the project.
+This survey form was created by Oscar and Habeeb detailing their experience with wildlife.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+- HTML
+- Css
+- Javascript
 
 
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/Efiamotu-1)
+- Twitter: [@twitterhandle](https://twitter.com/EFYAMOTU)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/musa-habeeb)
 
 👤 **Author2**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+Oscar
```

Comment:
The co-contributor's details and information should also be provided not just his name

Reply:
- [ ] [REQUIRED] Please, always try to mention the reason for your requested chagnes. Because it helps the students learn more.

Reply:
The co-contributor's details and information should also be provided not just his name because it helps future clients and other web developers easily get access to him/her for potential jobs, questions  and reviews on the project